Bit-Manipulation/SetBit.js

```diff
@@ -20,7 +20,7 @@
  * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_OR
  */
 
-/**
```

Comment:
This is a js doc comment, and it is supposed to start with `/**`. See [jsdoc](https://jsdoc.app/).